pkg/hintrunner/zero/zerohint_math_test.go

```diff
@@ -555,5 +555,79 @@ func TestZeroHintMath(t *testing.T) {
 				errCheck: errorTextContains("outside of the range [0, 2**250)"),
 			},
 		},
+
+		"SplitFelt": {
+			{
+				operanders: []*hintOperander{
+					{Name: "MAX_HIGH", Kind: apRelative, Value: &utils.FeltMax128},
+					{Name: "MAX_LOW", Kind: apRelative, Value: &utils.FeltZero},
```

Comment:
The values for `MAX_HIGH` and `MAX_LOW`, as mentioned in the previous comment are always the same:
```
// Note: the following code works because PRIME - 1 is divisible by 2**128.
const MAX_HIGH = (-1) / 2 ** 128;
const MAX_LOW = 0;
```
Same for the other test cases. Anyway, when you solve the previous comment this should be removed from the test case.

Reply:
As previously mentioned, removed it from testcases and fixed in methods